demos/sample_azure_iot_adu/sample_azure_iot_adu.c

```diff
@@ -964,6 +1006,7 @@ static void prvAzureDemoTask( void * pvParameters )
         TlsTransportParams_t xTlsTransportParams = { 0 };
         AzureIoTResult_t xResult;
         AzureIoTTransportInterface_t xTransport;
+        uint32_t ulOutProvisioningPayloadLength;
```

Comment:
```suggestion
        int32_t ulOutProvisioningPayloadLength;
```
AzureIoTJSONWriter_GetBytesUsed returns an int32_t